src/main/java/com/twilio/taskrouter/application/config/TaskRouterGuiceConfig.java

```diff
@@ -1,28 +1,20 @@
 package com.twilio.taskrouter.application.config;
 
 import com.google.inject.AbstractModule;
-import com.google.inject.name.Names;
+import com.twilio.taskrouter.domain.model.WorkspaceFacade;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Properties;
 import java.util.logging.Logger;
 
 /**
- * Provide the Workspace and related resources
+ * Provide the resources related to task router
```

Comment:
Our tutorials are extensions for the Twilio's documentation. You might want to take a look at https://www.twilio.com/taskrouter. Please use `TaskRouter` instead of `task router`.